packages/scandipwa/src/component/FieldNumber/FieldNumber.component.js

```diff
@@ -67,7 +67,7 @@ export class FieldNumber extends PureComponent {
                     <AddIcon block="SubtractButton" isPrimary />
                 </button>
                 <button
-                  disabled={ +value === min || isDisabled }
+                  disabled={ min === 1 + +value || +value === min || isDisabled }
```

Comment:
Expressions are evaluated from left to right.
```javascript
1 + +value
```
So `1 + ?` will always try to cast `?` to number. You don't need to cast `value` again here.

Reply:
Actually if would be more readable if you cast the `value` to the number before you make add and comparison operations.